design-documents/graph-ql/coverage/requisitionList.graphqls

```diff
@@ -0,0 +1,261 @@
+## Gift Card product type support will be added after the Product Implementation is developed
+type Query {
+    "The requisitionList query returns the contents of customer's  Requisition List"
+    requisitionList(
+        id: ID!
+    ): RequisitionList
+
+    "Export Requisition list"
+    exportRequisitionList(
+        "unqiue Id of Requisition list"
+        id: ID!
+    ): CsvOutput
```

Comment:
re: `CsvOutput`, I'm not quite convinced that it makes sense to stick a file download inside of what's basically a `String` type.

Doing some quick googling around, it seems like a common pattern for file downloads in GraphQL is to return a URL that's valid with the current session for 1 download.

Then the UI can use `<a href=${downloadLink} download>Download CSV</a>`, or `window.open(downloadLink)`

Reply:
Looks like this was discussed in https://github.com/magento/architecture/pull/378#issuecomment-629267899

@mbrinton01 would prefer not to inline file downloads into query responses. If we don't have the functionality to support file downloads, my preference would be to defer implementing export until we have a better solution.

Reply:
We can defer the export. It's not the most critical part of this feature and can be added in the future without breaking changes.

Reply:
removed export query from the schema document.